src/components/FullWidthCta.js

```diff
@@ -128,14 +130,12 @@ const FullWidthCta = props => {
                 {contentfulModuleToComponent({ ...embedHtml, previewMode })}
               </div>
             ) : null}
-            {ctas ? (
+            {!isEmpty(ctas) ? (
               <CTAWrapper>
-                {ctas.map(cta =>
-                  contentfulModuleToComponent({
-                    ...cta,
-                    previewMode,
-                  })
-                )}
+                {contentfulModuleToComponent({
+                  ...ctas[ctas.length > 1 && isMetaMaskInstalled ? 1 : 0],
```

Comment:
This module support multiple ctas in many places (Ex: The institutions page have this module with 3 ctas and does not depend on MetaMask extension)
So this code will break that, we need an extra condition

Reply:
Thank you for noticing this! I'll fix that

Reply:
Hey @viphan007 
Can you review the code again after my updates ? 
Sorry for asking you this twice

Reply:
Hi @kevin-chassagne, it worked properly